src/sentry/snuba/tasks.py

```diff
@@ -218,16 +218,26 @@ def _create_in_snuba(subscription: QuerySubscription) -> str:
             snuba_query,
             subscription.project.organization_id,
         )
-        # TODO: determine whether concatenating query_extra is proper
+        params = {
+            "organization_id": subscription.project.organization_id,
+            "project_id": [subscription.project_id],
+        }
+        skip_time_conditions = True
+        if subscription.timebox_start:
+            start = subscription.timebox_start.isoformat()
```

Comment:
Yea hmmm

So Activated Alert QuerySubscriptions will self delete themselves after their window has elapsed - so we _should_ never need to update the windows

Reply:
Wondering if now that this timebox is explicitly part of subscriptions, if we should auto clean up subscriptions with a timebox too, rather than waiting for another system to do it?